packages/contentful--create-contentful-app/src/includeAppAction.ts

```diff
@@ -1,62 +1,61 @@
 import inquirer from 'inquirer';
-import { readFileSync, writeFileSync, mkdirSync, existsSync } from 'fs';
+import { readFileSync, writeFileSync, existsSync, renameSync } from 'fs';
 import { resolve } from 'path';
 import { CONTENTFUL_APP_MANIFEST } from './constants';
+import degit from 'degit';
+import { success, highlight } from './logger';
 
-export function cloneAppAction(templateIsTypescript: boolean, destination: string) {
-  const actionPath = resolve(
-    templateIsTypescript
-      ? 'src/app-actions/typescript/index.ts'
-      : 'src/app-actions/javascript/index.js'
-  );
+export async function cloneAppAction(templateIsTypescript: boolean, destination: string) {
```

Comment:
Strange that non-primary boolean param goes first. Should be second and non-mandatory, I think (with fallback to JS IMO).

Reply:
I can switch the order, but I think it needs to be mandatory. If we're not explicit we run the risk of cloning js files into a typescript repo, which will cause problems for users. Wdyt?

Reply:
Maybe it's better to switch it from boolean to `'typescript' | 'javascript'` type to support other templates in the future?

Reply:
> If we're not explicit we run the risk of cloning js files into a typescript repo, which will cause problems for users.

Yes, makes sense